README.md

```diff
@@ -66,11 +70,20 @@ jobs:
 
 ### Buildpacks - Release
 
-Prepares a buildpack release by:
-* bumping the fixed version
-* updating changelogs
-* generating an aggregate changelog from all the changelogs
-* opening a PR against the repository with the modified files
+Prepares one or more buildpacks release by:
```

Comment:
I think this sentence needs adjusting, since it mentions preparing a release, but this workflow is the "perform a release" one?

Reply:
done